MantidQt/CustomInterfaces/src/Indirect/Elwin.cpp

```diff
@@ -496,6 +493,54 @@ void Elwin::updateRS(QtProperty *prop, double val) {
     backgroundRangeSelector->setMaximum(val);
 }
 
+/**
+ * Handles mantid plotting
+ */
+void Elwin::plotClicked() {
+
+  auto outputWs = m_elwinAlg->getPropertyValue("OutputInQ");
+  checkADSForPlotSaveWorkspace(outputWs, true);
+  plotSpectrum(QString::fromStdString(outputWs));
+
+  auto outputWsSquared = m_elwinAlg->getPropertyValue("OutputInQSquared");
+  checkADSForPlotSaveWorkspace(outputWsSquared, true);
+  plotSpectrum(QString::fromStdString(outputWsSquared));
+
+  auto elfWs = m_elwinAlg->getPropertyValue("OutputELF");
+  checkADSForPlotSaveWorkspace(elfWs, true);
+  plotSpectrum(QString::fromStdString(elfWs), 0, 9);
+
+  if (m_blnManager->value(m_properties["Normalise"])) {
+    auto eltWs = m_elwinAlg->getPropertyValue("OutputELT");
+    checkADSForPlotSaveWorkspace(eltWs, true);
+    plotSpectrum(QString::fromStdString(eltWs), 0, 9);
+  }
+}
+
+/**
+ * Handles saving of workspaces
+ */
+void Elwin::saveClicked() {
+
+  auto outputWs = m_elwinAlg->getPropertyValue("OutputInQ");
+  checkADSForPlotSaveWorkspace(outputWs, false);
+  addSaveWorkspaceToQueue(QString::fromStdString(outputWs));
+
+  auto outputWsSquared = m_elwinAlg->getPropertyValue("OutputInQSquared");
+  checkADSForPlotSaveWorkspace(outputWsSquared, true);
+  addSaveWorkspaceToQueue(QString::fromStdString(outputWsSquared));
+
+  auto elfWs = m_elwinAlg->getPropertyValue("OutputELF");
+  checkADSForPlotSaveWorkspace(elfWs, true);
+  addSaveWorkspaceToQueue(QString::fromStdString(elfWs));
+
+  if (m_blnManager->value(m_properties["Normalise"])) {
+    auto eltWs = m_elwinAlg->getPropertyValue("OutputELT");
+    checkADSForPlotSaveWorkspace(eltWs, true);
```

Comment:
The same happens here (in addition I think in this case we should be checking `checkADSForPlotSaveWorkspace(eltWs, false)`)